diskqueue.go

```diff
@@ -630,15 +644,20 @@ func (d *diskQueue) checkTailCorruption(depth int64) {
 }
 
 func (d *diskQueue) moveForward() {
+	// add bytes for the number of messages and the size of the message
+	readFileSize := int64(d.readMsgSize) + d.readPos + 12
```

Comment:
nit: Is `int64(d.readMsgSize) + d.readPos + 12` equivalent to  `d.nextReadPos + numFileMsgsBytes`?

Reply:
Unfortunately no, :( https://github.com/kev1n80/go-diskqueue/blob/TrackDiskSize/diskqueue.go#L397 This number is needed when we meet the file size limit, and in this case `d.nextReadPos` will be 0.

Reply:
Got it. Thanks.